src/main/java/domain/model/MatchResult.java

```diff
@@ -0,0 +1,33 @@
+package domain.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static constant.LottoConstants.MATCH_PRICES;
+
+public class MatchResult {
+    private Map<Integer, Integer> resultMap = new HashMap<>();
```

Comment:
```suggestion
    private final Map<Integer, Integer> resultMap = new HashMap<>();
```
resultMap에 할당할 일이 더 이상 없다면 final을 붙여 불변을 유지하면 좋겠습니다!